SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are only provided for last released version.
```

Comment:
I think it kind of implies that one "stable" branch is maintained here upstream and that's not accurate. Fixes go to the master branch directly. Backports to released versions (including the latest one) are handled downstream. I think this paragraph can be removed.